tests/ui.rs

```diff
@@ -315,8 +315,7 @@ fn main() -> Result<()> {
     Ok(())
 }
 
-fn run_dep_mode(target: String, mut args: impl Iterator<Item = OsString>) -> Result<()> {
-    let path = args.next().expect("./miri run-dep must be followed by a file name");
+fn run_dep_mode(target: String, args: impl Iterator<Item = OsString>) -> Result<()> {
```

Comment:
`path` was `--edition` 🙃 we have started adding more flags.

This was also just a refactoring leftover, as we always added `path` right before `args` and never used `path` elsewhere